docs/about/contributing/CONTRIBUTING.md

```diff
@@ -148,6 +148,32 @@ channel.
 
 - [Frontend styling guide](./frontend.md#styling)
 
+## Pull Requests
+
+We welcome pull requests (PRs) from community members including (but not limited to) open source users, enthusiasts, and enterprise customers.
+
+We will ask that you sign a Contributor License Agreement before we accept any contributions into our repo.
+
+Please keep PRs small and self contained. This allows code reviewers (see below) to focus and fully understand the PR. A good rule of thumb is less than 1000 lines changed. (One exception is a mechanistic refactor, like renaming, that is conceptually trivial but might have a large line count.)
+
+If your intended feature or refactor will be larger than this:
+
+ 1. Open an issue explaining what you intend to build, how it will work, and that you are volunteering to do the development.
+ 2. Give the maintainers a chance to respond. Changes to the visual, interaction, or software design are easier to adjust before you start laying down code.
+ 3. Break your work up into a series of smaller PRs.
+
+Stacking tools like [Graphite](https://www.graphite.dev) are useful for keeping a series of PRs that build on each other up to date as they are reviewed and merged.
+
+Each PR:
+
+- Must individually build and pass all tests, including formatting and linting.
+- Must not introduce regressions or back compatibility issues, even if a later PR would resolve the issue.
```

Comment:
```suggestion
- Must not introduce regressions or backwards-compatibility issues, even if a subsequent PR in your changeset would resolve the issue.
```

Reply:
I don't like using "changeset" in this way because it's a synonym for a single commit. https://en.wikipedia.org/wiki/Changeset

Maybe "subsequent PR in the series would resolve the issue"